### PR TITLE
Edicion

### PR DESCRIPTION
**Pull request by germangelv**
## Summary

- Added a GET / POST to edit the movie
    - src/Controllers/MovieController.cs
- Modified Repository to update the movie
    - src/Repositories/Repository.cs
- Created view that reuses code
    - src/Views/Movie/Edit.cshtml
    - src/Views/Movie/Create.cshtml
    - src/Views/Movie/_Form.cshtml
    - src/Views/Shared/_Layout.cshtml
- Added new globalizing elements
    - src/Resources/en-US.json
    - src/Resources/es-ES.json
    - src/Resources/fr-FR.json

## Trello Card

https://trello.com/c/Uz4eavFq
	
## Screenshot
 - URL error example

image
 - URL correct example

image

### CODE REVIEW

File: src/Controllers/MovieController.cs [hunk 1]
```diff
@@ -46,5 +47,57 @@ public IActionResult Create(MovieViewModel movieVM)
             }
             return View();
         }
+               
+        [HttpGet("Edit/{id}")]
+        public IActionResult Edit(int? id)
+        {
+            try 
+            {
+
```
Comment (truccomarcos) [inline src/Controllers/MovieController.cs]: Remove this empty line

File: src/Views/Movie/_Form.cshtml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            <div class = "col-md-10">
+                @Html.TextBoxFor(m => m.Price, new { @class = "form-control" })
+            </div>
+        </div>
+        <div class = "form-group">
+            <div class = "col-md-10">
+                <input type = "submit" class = "btn btn-default" value = @ViewData["ButtonOK"] />
+        </div>
+    </div>
+}
```
Comment (truccomarcos) [inline src/Views/Movie/_Form.cshtml]: Add an empty line here

File: src/Views/Movie/_Form.cshtml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <div class = "form-group">
+            @Html.Label("Title",Localizer["Movie"].Value, new { @class = "col-md-2 control-label" })
+            <div class = "col-md-10">
+                @Html.TextBoxFor(m => m.Title, new { @class = "form-control" })
+            </div>
+        </div>
+        <div class = "form-group">
+            @Html.Label("ReleaseDate",Localizer["ReleaseDate"].Value, new { @class = "col-md-2 control-label" })
+            <div class = "col-md-10">
+                @Html.TextBoxFor(m => m.ReleaseDate, new { @class = "form-control" })
```
Comment (truccomarcos) [inline src/Views/Movie/_Form.cshtml]: Try using EditorFor instead of TextBoxFor

Comment (germangelv) [reply src/Views/Movie/_Form.cshtml]: I will take your advice on the following Card

**Pull request merged**